packages/web3-core/src/web3_subscriptions.ts

```diff
@@ -82,16 +81,6 @@ export abstract class Web3Subscription<
 		super();
 		const { requestManager } = options as { requestManager: Web3RequestManager<API> };
 		const { subscriptionManager } = options as { subscriptionManager: Web3SubscriptionManager };
-		if (requestManager && subscriptionManager) {
```

Comment:
removed, in the [constructor](https://github.com/web3/web3.js/blob/c5d47f8c68f6b69114dc9708759d30449315c015/packages/web3-core/src/web3_subscriptions.ts#L63)
`requestManager` or `subscriptionManager` must be defined